### PR TITLE
Add LogMeIn and Mapbox icon

### PR DESCRIPTION
**Pull request by dpeukert**

### CODE REVIEW

**Comment by dpeukert** [comment]
Oops, haven't noticed the other Mapbox PR. I can remove the MB logo if you want.

File: app/src/main/res/drawable/thumb_logmein.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="446dp"
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_logmein.xml]: We try to keep thumbnails bellow 240dp

Comment (dpeukert) [reply app/src/main/res/drawable/thumb_logmein.xml]: Oops, I thought the shorter side was supposed to be <240dp. Will fix.

Comment (dpeukert) [reply app/src/main/res/drawable/thumb_logmein.xml]: Fixed.

**Comment by dpeukert** [comment]
Sorry about the commit name, I obviously meant LogMeIn thumb size.

**Comment by flocke** [comment]
@dpeukert I prefer your version of the MapBox icon (since it's really only the icon, not the whole logo as in #235). I'm gonna keep your version and close #235. As soon as @RichyHBM approves of your changes they will be merged.

**Pull request merged**